### ISSUE TITLE
Do you have any plan for supporing Windows Hyper-V?

### ISSUE THREAD

**Issue by wgtgithub** [pires/kubernetes-vagrant-coreos-cluster#269]
Hello,

I would like to try this package for Windows Hyper-V.
Or do you have any idea for me?

**Comment by pires** [comment]
I don't have Windows much less Windows Server.

**Comment by wgtgithub** [comment]
I see.
Thanks.

**Comment by dwickern** [comment]
I was able to get Hyper-V barely working but it's not easy:
- There's no official CoreOS vagrant box for Hyper-V (although I was able to get one working in https://github.com/dwickern/coreos-packer)
- CoreOS doesn't ship with Hyper-V daemons (without this, the guest will not report its IP)
- NFS shared folder isn't supported on Hyper-V (even with `vagrant-winnfsd`)
- SMB isn't supported by CoreOS (missing `/sbin/cifs.mount`)
- Rsync works but requires a manual step, and it only copies from host-to-guest
- Hyper-V doesn't support NAT at all, so you need to use a public network

In conclusion, Hyper-V _can_ work but will not work well. Don't bother.
